Dockerfile

```diff
@@ -1,16 +1,17 @@
 FROM node:alpine
 
-ARG APP_PORT
+ARG SERVER_PORT
 
 WORKDIR /app
 COPY . ./
 
 ENV PATH ./node_modules/.bin:$PATH
+ENV NODE_ENV "development"
 
 USER node
 
 RUN mkdir -p /app/node_modules
 
-EXPOSE $APP_PORT
+EXPOSE $SERVER_PORT
 
 CMD [ "sh", "-c", "mkdir -p /app/node_modules && npm ci && npm run dev" ]
```

Comment:
@IAmRGroot Why do we create the directory `/app/node_modules` again here? Can we simplify this command and/or remove the RUN command?

Also, does `npm ci` require the `node_modules` folder to be present? Or can we skip creating the directory all together like we can when running `npm i`?

Reply:
Because it otherwise won't work if you just cloned the repo, node_modules is hot ignored.

I think the command can be improved, but I'm not sure now to run npm ci only when needed

Reply:
From the npm docs:

> `npm ci` will be significantly faster when:
> 
> There is a `package-lock.json` or `npm-shrinkwrap.json` file.
> The `node_modules` folder is missing or empty.

So it should be fine to leave the `mkdir` statement(s) out? Guess I'll do just that and figure out if things still work on a clean install.

Reply:
I kinda fixed it in https://github.com/media-assistant/mass-vue/pull/14/commits/e66945dedcb85ccd069f4016c8377dfcc9ec5be7